autoload/merginal/buffers/historyLog.vim

```diff
@@ -12,8 +12,10 @@ function! s:f.generateHeader() dict abort
                 \ '']
 endfunction
 
+let g:merginal_log_commit_count = get(g:, 'merginal_log_commit_count', -20)
```

Comment:
* The setting should be named `g:merginal_logCommitCount` - this is the style I use in all my plugins.
* I don't like that set-a-default-somewhere global approach - it's not that hard to pick the default at the point of usage, and it allows the user to return to default with a simple `:unlet`.
* Using a negative number to feed it to the `-{number}` short style is just horrible. If a user wants to set this flag, the intuitive way would be `let g:merginal_logCommitCount = 100` - but this will break the history log and the user will not know why. Use the `--max-count` flag instead.